up.ps1

```diff
@@ -142,6 +142,8 @@ if ($LASTEXITCODE -ne 0) {
     Write-Error "Serialization push failed, see errors above."
 }
 
+Invoke-WebRequest https://$xmCloudHost/Utilities/Restart.aspx

```

Comment:
This will affect only local Docker instances. Is the cache clearing issue also in deployed instances?

Please wrap this line with DEMO TEAM CUSTOMIZATION comments to track our changes in this starter file. Thanks

Reply:
Must be a different issue for deployed instances, since those are using IaR, will investigate that one.

Reply:
Sorry. I did not say there was a caching issue on deployed instances. I asked a question to know whether there was one.

So is there a cache issue on deployed instances? Or is it only on local Docker environments?

Reply:
Yes, I see a similar issue on deployed one, still investigating.